dspace-api/src/test/java/org/dspace/authorize/AuthorizeServiceTest.java

```diff
@@ -129,70 +127,72 @@ public void testauthorizeMethodRespectSpecialGroups() {
             throw new AssertionError(ex);
         }
     }
-
-    @Test
-    public void testIsCollectionAdmin() throws SQLException, AuthorizeException, IOException {
-
-        Community community = null;
-        EPerson eperson = null;
-
-        try {
-
-            context.turnOffAuthorisationSystem();
-
-            community = communityService.create(null, context);
-            Collection collection = collectionService.create(context, community);
-            eperson = ePersonService.create(context);
-
-            Group administrators = collectionService.createAdministrators(context, collection);
-            groupService.addMember(context, administrators, eperson);
-            context.commit();
-
-            Assert.assertTrue(authorizeService.isCollectionAdmin(context, eperson));
-
-        } finally {
-
-            if (community != null) {
-                communityService.delete(context, context.reloadEntity(community));
-            }
-            if (eperson != null) {
-                ePersonService.delete(context, context.reloadEntity(eperson));
-            }
-
-            context.restoreAuthSystemState();
-        }
-    }
-
-    @Test
-    public void testIsCollectionAdminReturnsTrueIfTheUserIsCommunityAdmin()
-        throws SQLException, AuthorizeException, IOException {
-
-        Community community = null;
-        EPerson eperson = null;
-
-        try {
-
-            context.turnOffAuthorisationSystem();
-
-            community = communityService.create(null, context);
-            eperson = ePersonService.create(context);
-
-            Group administrators = communityService.createAdministrators(context, community);
-            groupService.addMember(context, administrators, eperson);
-            context.commit();
-
-            Assert.assertTrue(authorizeService.isCollectionAdmin(context, eperson));
-
-        } finally {
-
-            if (community != null) {
-                communityService.delete(context, context.reloadEntity(community));
-            }
-            if (eperson != null) {
-                ePersonService.delete(context, context.reloadEntity(eperson));
-            }
-
-            context.restoreAuthSystemState();
-        }
-    }
+//
+//    @Test
```

Comment:
@Raf-atmire : It looks like these tests were accidentally left commented out?  Was that purposeful?  If so, let's create a ticket to follow this PR up with a fix for these tests (if for some reason they currently don't work).

Reply:
I tried reenabling them, and sure enough they don't work right now, as they seem to assume the Database approach instead of the new Solr approach.  I will treat this as a *follow-up* task, rather than requiring it in this PR.  The new Solr approach to determining Community/Collection Admins already has ITs that prove it works. So, we can refactor these Unit Tests in a follow-up ticket.